src/metadata.ts

```diff
@@ -8,80 +10,39 @@ export interface IPageMetadata {
 const TITLE_PREFIX = 'FreeBoardGame.org - ';
 
 const DEFAULT_METADATA: IPageMetadata = {
-  title: TITLE_PREFIX + 'Play Free Board Games Online',
-  description: 'Play board games in your browser for free. \
-Compete against your online friends or play locally. Free and open-source software project.',
+  title: TITLE_PREFIX,
+  description: 'Free Board Game',
```

Comment:
Just "Free Board Game"?

Reply:
Yeah, these pages should not be crawlable, they have no index, so it doesnt make a difference